cloudbuild.yaml

```diff
@@ -15,22 +16,10 @@ steps:
   args:
   - 'buildx'
   - 'create'
+  - '--use' # Create and use builder in one step
```

Comment:
Same with these, what are they doing?

Reply:
Combining multi-arch steps 1,2,3 into 1. Where we create a builder for multi-arch implementation and bootstrap it for using in the multi-arch build steps below